streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsProducerTest.java

```diff
@@ -1098,20 +1053,16 @@ public void shouldSwallowExceptionOnEosAbortTxInvalidEpoch() {
     }
 
     private void testSwallowExceptionOnEosAbortTx(final RuntimeException exception) {
-        mockedProducer.initTransactions();
-        mockedProducer.beginTransaction();
-        expect(mockedProducer.send(record, null)).andReturn(null);
-        mockedProducer.abortTransaction();
-        expectLastCall().andThrow(exception);
```

Comment:
`mockedProducer.abortTransaction` was expected to throw an exception, this doesn't appear to be the case in the new version of this test - is there a reason for this?

Reply:
fixed.